Add more details on cli collisions (#968)
salesforcecli/cli: .github/workflows/v2-beta.md

## Installation

> **Warning**  
> You **must** uninstall `sfdx` before you install `sf` (v2), or you'll get an error. See instructions for your installation methods below.
> 

> > **Context:** We created an alias for `sfdx` within `sf` (v2) so that you would not have to update all of your scripts from `sfdx` to `sf`. The concequence is a one-time uninstall of `sfdx-cli` to prevent this collision.

Reviewer: ```suggestion
> **Context:** We created an alias for `sfdx` within `sf` (v2) so that you don't need to update your scripts from `sfdx` to `sf`. The consequence is a one-time uninstall of `sfdx-cli` to prevent this collision.
```

1. Remove the command to install `sfdx`.  
2. Add (or replace) the command that installs `sf` with this command:  `npm install -g @salesforce/cli@beta`.

> If you see an error similar to this, it means you need to uninstall `sfdx-cli`

Reviewer: ```suggestion
If you see an error similar to this, you must uninstall `sfdx-cli`.
```
